src/storage/src/sst.rs

```diff
@@ -91,6 +93,15 @@ impl LevelMetas {
         self.levels.len()
     }
 
+    pub fn set_compaction_time_window(&self, window: i64) {
```

Comment:
Maybe we could update the window in `merge()`. Like how we update `flushed_sequence` in `apply_edit`
```rust
pub fn merge(
    &self,
    files_to_add: impl Iterator<Item = FileMeta>,
    files_to_remove: impl Iterator<Item = FileMeta>,
    compaction_window: Option<i64>,
) -> LevelMetas {}
```